qiita_pet/templates/change_lost_pass.html

```diff
@@ -8,7 +8,7 @@
 {%end%}
 
 {% block content %}
-  {% if not message %}
+  {% if display_body %}
```

Comment:
I'm at dinner and don't know the specific tests off the top of my head. I just wanted to make sure that this was brought up before the request was merged by two other devs.